packages/driver/src/reflection/index.ts

```diff
@@ -30,4 +30,4 @@ export * from "./reservedKeywords";
 // export * from "../syntax/funcops";
 // export * from "./syntax";
 export * as introspect from "./queries";
-export * from "./analyzeQuery";
+export { analyzeQuery } from "./analyzeQuery";
```

Comment:
Was this change necessary or are you being defensive here to avoid leaking the new exports?

Reply:
Defensive. But I forgot until now that reflection/index isn't exported at top level. So many exporting these functions just to reflection is ok. Thoughts?

Reply:
I don't have a strong opinion on this, but I guess it's safest to be restrictive now and add more to the export if we want them in the future, so good to keep what you have here.

Reply:
Ok I adjusted names to better fit a more generalized export/import.

The more I look at this the more I wonder why this is in the driver package and not the generation one...